.github/workflows/run-tests.yaml

```diff
@@ -61,46 +61,3 @@ jobs:
         run: |
           rm -f ~/.gradle/caches/modules-2/modules-2.lock
           rm -f ~/.gradle/caches/modules-2/gc.properties
-  accessibility-test:
-    name: Run accessibility tests
-    runs-on: ubuntu-latest
-    permissions:
-      actions: read
-    services:
-      postgres:
-        image: postgres
-        env:
-          POSTGRES_USER: mdbenefits_test
-          POSTGRES_DB: mdbenefits_test
-          POSTGRES_PASSWORD: postgres
```

Comment:
Do we want to take these out or just fix them?

Reply:
Never mind. We can add this logic back in if we need to. The tests really do need to go as they don't apply to our setup at all.